tools/iproxy.c

```diff
@@ -297,6 +297,7 @@ int main(int argc, char **argv)
 		int c_sock;
 		printf("waiting for connection\n");
 		while (1) {
+			printf("waiting for connection\n");
```

Comment:
Is this required ? I guess waiting for connection keeps on printing since it is infinite while loop